tasks.py.jinja

```diff
@@ -1,4 +1,336 @@
+        },
+        "skipFiles": ["**/lib/**"],
+        "log": {"consoleLevel": {"PathConversion": "Debug"}},
+        "pathMappings": [],
+    }
+    chrome_executable = which("chrome")
+    chromium_executable = which("chromium")
+    chrome_configuration = {
+        "type": "chrome",
+        "request": "launch",
+        "preLaunchTask": "Start project",
+        "name": "Connect to chrome debugger",
+        "url": f"http://localhost:{ODOO_VERSION:.0f}069/?debug=assets",
+        "skipFiles": ["**/lib/**"],
+        "trace": True,
+        "pathMapping": {},
+    }
+    if chrome_executable:
+        chrome_configuration["runtimeExecutable"] = chrome_executable
+    elif chromium_executable:
+        chrome_configuration["runtimeExecutable"] = chromium_executable
+    cw_config["launch"] = {
+        "configurations": [
+            {
+                "name": "Run in debug mode and attach",
+                "type": "python",
+                "request": "attach",
+                "preLaunchTask": "Start project in debug mode",
+                "postDebugTask": "Stop project",
+                "pathMappings": [
+                    {
+                        "localRoot": f"{root_var}/odoo",
+                        "remoteRoot": "/opt/odoo",
+                    }
+                ],
+                "port": int(ODOO_VERSION)*1000 + 899,
+                "host": "localhost",
+            },
+            {
+                "name": "Attach to running container",
+                "type": "python",
+                "request": "attach",
+                "pathMappings": [
+                    {
+                        "localRoot": f"{root_var}/odoo",
+                        "remoteRoot": "/opt/odoo",
+                    }
+                ],
+                "port": int(ODOO_VERSION)*1000 + 899,
+                "host": "localhost",
+            },
+            firefox_configuration,
+            chrome_configuration,
+        ]
+    }
+    # Configure folders
+    cw_config["folders"] = []
+    for subrepo in SRC_PATH.glob("*"):
+        if (subrepo / ".git").exists():
+            cw_config["folders"].append({"path": str(subrepo)})
+        for addon in chain(subrepo.glob("*"), subrepo.glob("addons/*")):
+            if (addon / "__manifest__.py").is_file() or (
+                addon / "__openerp__.py"
+            ).is_file():
+                firefox_configuration["pathMappings"].append(
+                    {
+                        "url": f"http://localhost:{ODOO_VERSION:.0f}069/{addon.name}/static/",
+                        "path": "${workspaceRoot:%s}/%s/static/" % (subrepo.name, addon.relative_to(subrepo)),
+                    }
+                )
+                chrome_configuration["pathMapping"][
+                    f"http://localhost:{ODOO_VERSION:.0f}069/{addon.name}/static/"
+                ] = "${workspaceRoot:%s}/%s/static/" % (subrepo.name, addon.relative_to(subrepo))
```

Comment:
Since you're using the same strings twice, I think it's better to use a variable. It'll become more readable too:

```suggestion
                url = f"http://localhost:{ODOO_VERSION:.0f}069/{addon.name}/static/"
                path = "${workspaceRoot:%s}/%s/static/" % (
                    subrepo.name,
                    addon.relative_to(subrepo),
                )
                firefox_configuration["pathMappings"].append({"url": url, "path": path})
                chrome_configuration["pathMapping"][url] = path
```